README.md

```diff
@@ -1,30 +1,34 @@
-# nwoc_algorithms
-This is a library of various algorithms that are used in competitive programming.
-
-
-### Sample algorithm
-[disjoint_set_union.cpp](Algorithms/disjoint_set_union.cpp) <br />
-[Floyd_cycle_detection.cpp](Algorithms/Linked_List/Floyd_cycle_detection.cpp)
-
-### Bellman Ford Algorithm
- [Bellman Ford Algorithm](Bellman_Ford/BellmanFord.cpp)
-
-### Sorting algorithm 
-[Merge Sort](Algorithms/Sorting/merge_sort.cpp) <br/>
-[Heap Sort](Algorithms/Sorting/heap_sort.cpp)<br/>
-[Insertion Sort](Algorithms/Sorting/insertion_sort.cpp) <br/>
-[Bubble Sort](Algorithms/Sorting/bubble_sort.cpp) <br/>
-[Selection Sort](Algorithms/Sorting/selection_sort.cpp)
-
-### String Hashing
-[Rabin Karp Algorithm](Algorithms/String_Hashing/rabin_karp.cpp)<br/>
-[Count_Substring](Algorithms/String_Hashing/count_substr.cpp) <br/>
-[Z Algorithm](Algorithms/String_Hashing/z-algorithm.cpp)<br/>
-[Manacher Algorithm](Algorithms/String_Hashing/manacher.cpp) <br/>
-
-### Backtracking Algorithms
-[Hamiltonian Cycle](https://github.com/SubhradeepSS/nwoc_algorithms/blob/master/Algorithms/Backtracking/Hamiltonian%20Cycle.cpp)<br/>
-[M Coloring Problem](https://github.com/SubhradeepSS/nwoc_algorithms/blob/master/Algorithms/Backtracking/M%20Coloring%20Problem.cpp)<br/>
-[N Queen Problem](https://github.com/SubhradeepSS/nwoc_algorithms/blob/master/Algorithms/Backtracking/N%20Queen%20Problem.cpp)<br/>
-[Rat in a Maze](https://github.com/SubhradeepSS/nwoc_algorithms/blob/master/Algorithms/Backtracking/Rat%20in%20a%20Maze.cpp)<br/>
-[Sudoku](https://github.com/SubhradeepSS/nwoc_algorithms/blob/master/Algorithms/Backtracking/Sudoku.cpp)
+# nwoc_algorithms
+This is a library of various algorithms that are used in competitive programming.
+
+
+### Sample algorithm
+[disjoint_set_union.cpp](Algorithms/disjoint_set_union.cpp) <br />
+[Floyd_cycle_detection.cpp](Algorithms/Linked_List/Floyd_cycle_detection.cpp)
+
+### Bellman Ford Algorithm
+ [Bellman Ford Algorithm](Bellman_Ford/BellmanFord.cpp)
+(Algorithms/.cpp)
+
+### Greedy Algorithm
+[Egyptian Fraction](https://github.com/NJACKWinterOfCode/nwoc_algorithms/pull/116/commits/a660bb5ca89951b744ae27ebc808c6e032ea6053)(Algorithms/EgyptianFractions.cpp)

```

Comment:
This does not give link
Look in rich text

Reply:
I have updated readme now with correct link so please review it. I think it will work fine now